saleor/core/management/commands/change_currency.py

```diff
@@ -0,0 +1,47 @@
+from babel.numbers import UnknownCurrencyError, validate_currency
+from django.core.management.base import BaseCommand, CommandError
+
+from ....checkout.models import Checkout
+from ....discount.models import Voucher
+from ....giftcard.models import GiftCard
+from ....order.models import Order, OrderLine
+from ....payment.models import Payment, Transaction
+from ....product.models import Product, ProductVariant
+from ....shipping.models import ShippingMethod
+
+
+class Command(BaseCommand):
+    help = "Change currency in all models in database."
```

Comment:
"Change currency in all models in the database. Note, that this command only changes currency code without doing any conversion. Currency set by this command must match with the value set in `DEFAULT_CURRENCY` environment variable."